packages/react-frontend/src/pages/NewQuestion.js

```diff
@@ -37,7 +36,6 @@ function NewQuestion() {
 
   return (
     <>
-      <Navbar navbar>navbar</Navbar>
```

Comment:
Why was this removed?

Reply:
It was incorrectly in the NewQuestion.js file so I removed it.